include/ccl_mass_conversion.h

```diff
@@ -0,0 +1,23 @@
+/** @file */
+#ifndef __CCL_M_CONV_H_INCLUDED__
+#define __CCL_M_CONV_H_INCLUDED__
+
+CCL_BEGIN_DECLS
+
+/**
+ * Get concentration for a new mass definition.
+ * @param cosmo Cosmological parameters
+ * @param delta_old overdensity parameter in the input mass definition.
+ * @param nc number of concentration values to translate.
+ * @param c_old input concentration.
+ * @param delta_new overdensity parameter in the output mass definition.
+ * @param c_new output concentration.
+ * @param status Status flat. 0 if everything went well.
+ */
+void ccl_get_new_concentration(ccl_cosmology *cosmo,
```

Comment:
The name of this function is not great. This is a conversion utility right? Maybe it should be called something like that?

Reply:
changed to `convert_concentration`